go.mod

```diff
@@ -79,7 +79,6 @@ require (
 )
 
 replace (
-	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220214200702-86341886e292 // CVE-2021-43565
-	k8s.io/client-go => k8s.io/client-go v0.23.3
-	open-cluster-management.io/governance-policy-propagator => github.com/stolostron/governance-policy-propagator v0.0.0-20220427184903-387712d230ee
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
```

Comment:
Can you add the CVE comment back so we remember why this is replaced?

Reply:
Updated